src/main/java/com/palantir/stash/stashbot/servlet/RepoConfigurationServlet.java

```diff
@@ -113,6 +113,11 @@ public void doGet(HttpServletRequest req, HttpServletResponse res) throws Servle
                 ImmutableMap.Builder<String, String> m = ImmutableMap.builder();
                 m.put("text", jsc.getName());
                 m.put("value", jsc.getName());
+                log.info("Printing possible causes of a NullPointerException: \n" +
+                    "    jsc: " + String.valueOf(jsc) + "\n" +
+                    "    jsc.getName(): " + String.valueOf(jsc.getName()) + "\n" +
```

Comment:
Is it possible that `jsc` and `rc` can be null? If so, I don't think this code as written will log anything—it would blow up with a NPE.

Reply:
Based on the context and usage of `jsc` and `rc` prior to the log statement, I don't think they are null. However, just to be sure, I added a null check for both variables prior to the log statements that define a default value for the `getName` and `getJenkinsServerName` methods.